Sources/SKSwiftPMWorkspace/SwiftPMWorkspace.swift

```diff
@@ -293,6 +293,13 @@ extension SwiftPMWorkspace: SKCore.BuildSystem {
 
     return nil
   }
+
+  public func filesDidChange(_ events: [FileEvent]) {
+    if events.contains(where: { $0.type == .created }) {
+      // TODO: It should not be necessary to reload the entire package just to get build settings for one file.
```

Comment:
Is this true? I'm not sure exactly how the SPM integration works, is there some incremental reload that we could do?

Reply:
I’m not sure if it’s currently possible with the API SwiftPM provides, but in principle it shouldn’t be necessary to reload the entire package just to get the build settings for one new file.